indexer/workers/importer.py

```diff
@@ -165,7 +166,11 @@ def process_story(self, sender: StorySender, story: BaseStory) -> None:
 
             # if publication date is none, fallback to rss_fetcher pub_date
             if data["publication_date"] is None:
-                data["publication_date"] = story.rss_entry()["pub_date"]
+                pub_date = story.rss_entry().pub_date
+                if pub_date is not None and pub_date != "":
+                    tzinfos = {"UTC": tz.UTC} 
+                    es_date = parser.parse(pub_date, tzinfos=tzinfos).isoformat()
```

Comment:
```suggestion
                    es_date = parser.parse(pub_date)
```
`parse` should understand `UTC`, `GMT`, `Z` by [default](https://dateutil.readthedocs.io/en/stable/parser.html#classes).
According to Phil's comment [here](https://github.com/mediacloud/story-indexer/issues/172#issuecomment-1902514227), it's possible for `pub_date` to have a missing timezone. We may need to check if `tzinfo` exists before calling `isoformat`